### DISCUSSION TITLE
Is there any way to remove past event, but not today's events?

### DISCUSSION THREAD

**Discussion by rtromao** [derekantrican/GAS-ICS-Sync#354] [Q&A] [answered]
Hi,
I just tried to change some code, but my programing skills lack so much.

I would like to remove past events from the target calendar but keep today's events. Like something remove day - 1 events.

Tried to play with the date functions to force the current date to be -24 hours, but I could not.

Can someone give me a hint?

Thank you so much

**Comment by Lonestarjeepin** [comment, marked as answer]
https://github.com/derekantrican/GAS-ICS-Sync/issues/183

I think that will get you what you are looking for.  This example uses -90, but you can change it to -1.

**Reply by rtromao** [reply]
This is exactly what I was looking for. Thank you so much.
